src/Data/Models/ChannelOperationRequest.cs

```diff
@@ -150,11 +150,6 @@ public decimal Amount
         [NotMapped]
         public int NumberOfSignaturesCollected => ChannelOperationRequestPsbts == null ? 0 : ChannelOperationRequestPsbts.Count(x => !x.IsFinalisedPSBT && !x.IsTemplatePSBT && !x.IsInternalWalletPSBT);
 
-        /// <summary>
-        /// This is the JobId provided by Quartz of the job executing this request.
-        /// </summary>
-        public string? JobId { get; set; }
-
```

Comment:
Wait, why do we remove this then? Its actually used, though mainly debug @RodriFS